### ISSUE TITLE
Buttons for flag all selected samples or just only the actually shown

### ISSUE THREAD

**Issue by avelo** [ocean-data-qc/ocean-data-qc#3]
It could be nice to have 2 flag buttons, one for just flagging the actual sample and other for flagging the selected group.

**Comment by chesucr** [comment]
You are right right. In the meantime, remember that you can change the flag value of a single sample in the table below. You just need to look the field and update the flag.
